### DISCUSSION TITLE
Compiled version

### DISCUSSION THREAD

**Discussion by AutumnalBlake** [DigitalLogicSimCommunity/Digital-Logic-Sim-CE#52] [General]
Hi, would it be possible to get a compiled version here?

**Comment by AOx0** [comment]
There are already compiled versions in each [release](https://github.com/DigitalLogicSimCommunity/Digital-Logic-Sim-CE/releases/latest)
<img width="1241" alt="Screenshot 2023-01-04 at 11 42 48 p m" src="https://user-images.githubusercontent.com/50227494/210709952-e1beb9c7-ad5e-4c04-a165-f1e62ac963a1.png">

**Reply by AutumnalBlake** [reply]
Ahhh thanks!!

**Reply by AOx0** [reply]
Have a nice day :D
